src/app/quotations/page.tsx

```
@@ -101,7 +103,7 @@ export default function QuotationsPage() {
 
   useEffect(() => {
     loadQuotations()
-  }, [])
+  }, [loadQuotations])
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)